src/pages/nft/claim/index.vue

```diff
@@ -30,39 +30,6 @@
         :is-content-viewable="true"
         @view-content-url="handleClickViewContentDirectly"
       />
-      <template v-if="isFreePurchase">
-        <template v-if="state === 'INITIAL'">
-          <Label
-            class="text-medium-gray"
-            preset="p6"
-            :text="$t('nft_free_claim_email_label')"
-            align="center"
-          />
-          <div class="flex w-full py-[10px] px-[16px] gap-[12px] bg-shade-gray rounded-[12px]">
-            <input
-              v-model="claimingFreeEmail"
-              class="w-full bg-transparent border-0 focus-visible:outline-none"
-              :placeholder="$t('nft_free_claim_enter_email')"
-              type="email"
-            >
-          </div>
-          <ButtonV2
-            class="self-center mt-[24px]"
-            :text="$t('nft_free_claim_claim')"
-            preset="secondary"
-            @click="startFreePurchase"
-          />
-        </template>
-        <ProgressIndicator v-else-if="state === 'CLAIMING'" class="self-center" />
-        <template v-else-if="state === 'CLAIMED'">
-          <Label
-            class="text-medium-gray"
-            preset="p6"
-            :text="$t('nft_free_claim_check_email_label')"
-            align="center"
-          />
-        </template>
-      </template>
       <template v-else-if="!claimingAddress">
```

Comment:
Should be this cause of the missing claim button
```suggestion
      <template v-if="!claimingAddress">
```